sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/text/TextIOIT.java

```diff
@@ -54,7 +54,7 @@
  *
  * <p>Run this test using the command below. Pass in connection information via PipelineOptions:
  * <pre>
- *  mvn -e -Pio-it verify -pl sdks/java/io/text -DintegrationTestPipelineOptions='[
+ *  mvn -e -Pio-it verify -pl sdks/java/io/file-based-io-tests -DintegrationTestPipelineOptions='[
```

Comment:
How can we generalize this to support additional tests under 'file-based-io-tests' ?

Reply:
This points to whole `file-based-io-tests` module, so it should execute all tests within this module, right?

Reply:
This will run all tests under 'file-based-io-tests', no ? Please give the instructions here for just running the TextIOIT test.